deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -118,7 +118,7 @@ spec:
     Supported operating systems:
       - RHEL 7.x (x86 architecture)
 
-    > **Note**: The operator must be installed in the `kube-system` project.
+    > **Note**: The operator must be installed in the `default` project.
```

Comment:
forgot to mention that we are not allowed to change old CSV yamls

Reply:
done